tests/serial/map-if-necessary/two-solvers/mixed-substeps/ParallelImplicit.cpp

```diff
@@ -0,0 +1,31 @@
+#ifndef PRECICE_NO_MPI
+
+#include "../helper.hpp"
+#include "testing/Testing.hpp"
+
+#include <precice/precice.hpp>
+#include <vector>
+
+BOOST_AUTO_TEST_SUITE(Integration)
+BOOST_AUTO_TEST_SUITE(Serial)
+BOOST_AUTO_TEST_SUITE(MapIfNecessary)
+BOOST_AUTO_TEST_SUITE(TwoSolvers)
+BOOST_AUTO_TEST_SUITE(MixedSubsteps)
+
+BOOST_AUTO_TEST_CASE(ParallelImplicit)
+{
+  PRECICE_TEST("One"_on(1_rank), "Two"_on(1_rank));
+
+  std::vector<int> readMappings{1, 1, 1, 1, 1, 1};
```

Comment:
When moving to the next time window, there is no sample to map
```suggestion
  std::vector<int> readMappings{1, 0, 1, 0, 1, 0};
```

Reply:
I am not sure I get this one. The end of the window should be new when moving to the next time window, no?